docs/concepts.rst

```diff
@@ -415,13 +415,44 @@ explictly specify a serialization strategy for an entity by attaching a
 
 .. _Protocol: api/protocols.rst
 
-Exporting Persisted Files
-.........................
+Retrieving Persisted Files
```

Comment:
This prose is very nice and clear -- however, I feel like some of it can be left to other parts of the documentation.  It feels a bit like a self-contained explanation of all of `Flow.get`, whereas I think we can just focus on the key idea that it's possible to access persisted files.

In particular: at this point in the docs, the reader should already be aware of the existence of `get`, but may not have encountered multiplicity.  Also, we don't need to explain the full range of `mode`'s arguments and capabilities here -- we just need to make the reader aware of possibilities and link to `get`'s documentation, which should cover all the details.

What do you think?

Reply:
Yeah, happy to remove the parts about `collection` and `mode`'s arguments / capabilities.  Do you think including documentation about `FileCopier` is excessive?

Reply:
In this particular section of the docs, I think it can make sense to show that it's possible (like `flow.get('subject', mode='FileCopier').copy(...)`).  For the details on `FileCopier`, maybe the class should have docs in the API Reference section.

Reply:
Cool, will add.  Just to follow the naming conventions for docs in /docs/api, I'll make a `util.rst`.  Will just have `FileCopier` in it for now.